src/imitation/policies/serialize.py

```diff
@@ -96,23 +98,10 @@ def _load_stable_baselines(
         A function loading policies trained via cls.
     """
 
-    def get_model_path(path: str) -> str:
-        """Returns the model file in a path directory."""
-        model_path_pkl = os.path.join(path, "model.pkl")
-        model_path_zip = os.path.join(path, "model.zip")
-        assert (
-            os.path.exists(model_path_pkl) + os.path.exists(model_path_zip) == 1
-        ), f"Expected one model but found two:\n- {model_path_pkl}\n- {model_path_zip}"
-        if os.path.exists(model_path_pkl):
-            model_path = model_path_pkl
-        if os.path.exists(model_path_zip):
-            model_path = model_path_zip
-        return model_path
-
     def f(path: str, venv: VecEnv) -> BasePolicy:
         """Loads a policy saved to path, for environment env."""
         logging.info(f"Loading Stable Baselines policy for '{cls}' " f"from '{path}'")
-        model_path = get_model_path(path)
+        model_path = os.path.join(path, "model.pkl")
```

Comment:
Should this be `model.pt`? (I'm not certain)

Reply:
I'm not sure where the save code is that corresponds to this load code. The save code that I wrote for the bc and dagger classes only uses `checkpoint` and `policy` in filenames, e.g. `checkpoint-latest.pt` and `policy-latest.pt`. So this shouldn't be `model.pt` on account of the save code I wrote. At the moment, this loading code is loading the `model.pkl` from the zip file of the trained expert you sent me.

Reply:
Okay, thanks for clarifying!